api/host.go

```diff
@@ -0,0 +1,69 @@
+// Package api only includes constants and interfaces. This ensures no package cycles in implementation.
```

Comment:
oh I see nvm the comment ^^

Reply:
incidentally I think we can temporarily put this in the "wasm" package

Reply:
changed to wasm package for now as I think even with top-level entry points in the wazero package we'll still be able to keep packages for things like constants if we choose to. I'll show what I mean post-merge in #238